bullet_train/docs/stylesheets.md

```diff
@@ -0,0 +1,18 @@
+# Style Sheets
+Bullet Train's stock UI theme, called “Light”, is built to use `tailwindcss` extensively.
+
+As such, there are two ways to update the styling of your app: either by modifying theme partials, or by adding your own custom CSS.
+
+## Modify Theme Partials
+
+Since `tailwindcss` is used, most style changes are done by ejecting theme partials into a local copy, and modifying the Tailwind classes within their HTML.
```

Comment:
I think we can clarify this a little bit.

```suggestion
Since `tailwindcss` is used, most style changes are done by ejecting theme partials into your app's `app/views` directory, and modifying the Tailwind classes within the `.html.erb` templates.
```